dotcom-rendering/src/web/lib/LiveBlogRenderer.tsx

```diff
@@ -76,6 +97,7 @@ export const LiveBlogRenderer = ({
 							switches={switches}
 							isAdFreeUser={isAdFreeUser}
 							isSensitive={isSensitive}
+							isPinnedPost={true}
```

Comment:
This is the non pinned version so I believe it should be false

```suggestion
							isPinnedPost={false}
```

Reply:
Yep you're right - amended.